public/app/features/dashboard/dashgrid/DashboardGrid.tsx

```diff
@@ -249,9 +255,14 @@ export class DashboardGrid extends PureComponent<Props, State> {
                   onResizeStop={this.onResizeStop}
                   onLayoutChange={this.onLayoutChange}
                 >
+                  {/* // todo: if there are no panels, show empty page */}
```

Comment:
Is the comment referring to this behavior?

https://user-images.githubusercontent.com/239999/227004097-6a4814b1-3de5-46ee-9178-14a28092bffc.mp4

Reply:
oh, sorry, this is forgotten - should be deleted